resources/migrations/000_migrations.yaml

```diff
@@ -8049,7 +8049,7 @@ databaseChangeLog:
         - sql:
               sql: >-
                 UPDATE metabase_field
-                set effective_type    = 'type/DateTime',
+                set effective_type    = 'type/Instant',
```

Comment:
do you need another checksum on 288 in addition to 289?

Reply:
I would have thought so, but it didn't complain... not sure why

Reply:
went ahead and added one anyway